docs/man_pages/publishing/appmanager-upload-android.md

```diff
@@ -14,6 +14,7 @@ WARNING: This command is not applicable to mobile website projects. To view the
 * `--certificate` - Sets the certificate that you want to use for code signing your Android app. You can set a certificate by index or name. <% if(isHtml) { %>To list available certificates, run `$ appbuilder certificate`<% } %> 
 * `--download` - If set, downloads the application package to the root of the project.
 * `--publish` - If set, the application package will be automatically published and visible for the members of its distribution groups. To add distribution groups, use `--groups` option.
+* `--public` - If set, the published application will be available for download without requiring AppManager user credentials.
```

Comment:
Message for when the upload is complete, the app is **published** and we get the install URI:

Your app has been published successfully and is now available to download and install from: \<URL\>